lib/helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Helper
+class Helper
```

Comment:
I don't like name helper, it's too generit. Config is better in my test, it's generic but a little bit narrower